Dockerfile

```diff
@@ -27,6 +27,7 @@ COPY --from=builder /go/src/app/bundle-sync ./bundle-sync
 COPY resources ./resources
 COPY apispec ./apispec
 COPY bundles ./bundles
+COPY test_data ./test_data
```

Comment:
oh, yeah I meant to revisit this. I was having trouble running the image but it was because I wasn't properly importing my keys so it was looking for the default test keys in `test_data`. I can remove this

- [ ] remove import of test data